MamboBryan/tic-tac-toe
# #165100178 Add main activity UI

#### What does this PR do?
It adds the Main Activity which the user plays in and also its UI
#### Description of Task to be completed?
Have the following
- A new activity 
- The app name at the top of the activity
- A forfeit button in case the user wants to quit the match
- Forfeit button returns you to the Landing Activity
#### How should this be manually tested?
After cloning the repo open it in android studio
- Build and run the project to your preferred device
- Click the play with friend button and it should take you to a new activity
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/165100178
